### PR TITLE
add checksum joke

### PR DESCRIPTION
**Pull request by rfair404**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ Unfortunately these jokes only work if you git them.
 
 ---
 
+**Q:** Why did the security conscious engineer refuse to pay his dinner bill?
+
+**A:** Because he couldn't verify the checksum.
```
Comment (styfle) [inline readme.md]:
```suggestion
**A:** Because they couldn't verify the checksum.
```

File: readme.md [hunk 2]
```diff
@@ -20,6 +20,12 @@ Unfortunately these jokes only work if you git them.
 
 ---
 
+**Q:** Why did the security conscious engineer refuse to pay his dinner bill?
```
Comment (styfle) [inline readme.md]:
```suggestion
**Q:** Why did the security conscious engineer refuse to pay their dinner bill?
```

File: readme.md [hunk 3]
```diff
@@ -20,6 +20,12 @@ Unfortunately these jokes only work if you git them.
...
+
+**A:** Because he could not verify the checksum.
```
Comment (styfle) [inline readme.md]:
```suggestion
**A:** Because they could not verify the checksum.
```

File: readme.md (hunk 2 above)
Comment (rfair404) [reply readme.md]: good point!

File: readme.md (hunk 3 above)
Comment (rfair404) [reply readme.md]: also good point

**Pull request merged**